setup.py

```diff
@@ -63,6 +63,7 @@
         "rich>=10",
         "rich-click",
         "spectra>=0.0.10",
+        "seaborn",
```

Comment:
It looks like this new dependency is used to generate colour palettes.

MultiQC already has a bunch of functionality built in to do this, see https://github.com/ewels/MultiQC/blob/master/multiqc/utils/mqc_colour.py

Please use that instead and remove seaborn.

Reply:
we will use mpc colors instead.

Reply:
Thank you!